src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -14,38 +14,47 @@
 import edu.wpi.first.wpilibj.DriverStation.Alliance;
 import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.subsystem.IntakeSubsystem.IntakeConstants.IntakeMotorState;
+import frc.team2412.robot.subsystem.IntakeSubsystem.IntakeConstants.IntakeSolenoidState;
 import frc.team2412.robot.util.MultiplexedColorSensor;
+import io.github.oblarg.oblog.Loggable;
+import io.github.oblarg.oblog.annotations.Log;
 
-public class IntakeSubsystem extends SubsystemBase {
+public class IntakeSubsystem extends SubsystemBase implements Loggable {
 
     // Constants
 
     public static class IntakeConstants {
 
         public static Alliance teamColor = DriverStation.getAlliance();
 
-        public static Color BLUE_CARGO_COLOR = new Color(0, 0, 1);
-        public static Color RED_CARGO_COLOR = new Color(1, 0, 0);
+        public static final Color BLUE_CARGO_COLOR = new Color(0.0, 0.4, 0.7019607844);
```

Comment:
Consider moving these constants into hardware.java to all subsystems can use them.